rust/1.58.1/Dockerfile

```diff
@@ -51,7 +51,8 @@ RUN set -e \
 # Rust deps
 RUN set -e \
 	&& rustup component add llvm-tools-preview rustfmt clippy \
-	&& cargo install --git https://github.com/paritytech/cachepot
+	&& cargo install --git https://github.com/paritytech/cachepot \
+	&& cargo install rustfilt
```

Comment:
Should 1.56 version get that too or get removed entirely?

Reply:
Not sure about that. I'd prefer to leave older container recipes unmodified for history. Wdyt?

Reply:
Sure, no strong opinion here: either way, we will bump rustc eventually and if nobody needs demangling now, it can wait.